tests/console/wicked_before_test.pm

```diff
@@ -13,13 +13,13 @@
 use base "consoletest";
 use strict;
 use testapi;
-use utils qw(systemctl setup_static_network is_sle sle_version_at_least);
+use utils qw(firewall systemctl setup_static_network is_sle sle_version_at_least);
```

Comment:
Travis is complaining because the function is not there.

You put it on opensusebasetest.